src/System.Net.Sockets/tests/FunctionalTests/UnixDomainSocketTest.cs

```diff
@@ -1,249 +0,0 @@
-// Licensed to the .NET Foundation under one or more agreements.
-// The .NET Foundation licenses this file to you under the MIT license.
-// See the LICENSE file in the project root for more information.
-
```

Comment:
I see.  You're absolutely right that the Pipes stuff is giving more coverage now - but it's nice to have *some* coverage in the Sockets test, as those are the ones I'm most likely to run while trying changes to Sockets.  That said, I'm not sure it's very likely that a Sockets change would break Unix-domain sockets, but not anything else we're already testing...so I guess I'm ok with removing these for now.